content/en/docs/collector/architecture.md

```diff
@@ -44,21 +44,18 @@ The following diagram represents a typical pipeline:
 title: Pipeline
 ---
 flowchart LR
-    R1(Receiver 1) --> P1[Processor 1]
-    R2(Receiver 2) --> P1
-    RM(...) ~~~ P1
-    RN(Receiver N) --> P1
-    P1 --> P2[Processor 2]
-    P2 --> PM[...]
-    PM --> PN[Processor N]
-    PN --> FO((fan-out))
-    FO --> E1[[Exporter 1]]
-    FO --> E2[[Exporter 2]]
-    FO ~~~ EM[[...]]
-    FO --> EN[[Exporter N]]
-
-    %% The stroke color matches the website header.
-    classDef default fill:#e3e8fc,stroke:#4f62ad
```

Comment:
/cc @tiffany76, since you chose these original color overrides if I'm not mistaken.

Reply:
FYI, we're still quite close to brand colors for most of the diagrams. (I might have a solution to get rid of the yellow, but it'll take more work.)